docs/setup/compile.md

```diff
@@ -94,7 +97,8 @@ pipenv --python 3.7
 5. Install the PySpark version and other dependency
 ```
 cd python
-pipenv install pyspark==3.0.1
+pipenv install pyspark
```

Comment:
Maybe add some tips here to tell the users that this will always install the latest PySpark version. The user needs to double check the downloaded Spark in SPARK_HOME is consistent with the installed PySpark version.